### ISSUE TITLE
Enable ELB's in Amazon profile.

### ISSUE THREAD

**Issue by snoby** [CiscoCloud/mantl#1826]
This ticket ( #1688 )and the decision to remote the ELB's from ticket #1710. Puts myself as a user of mantl into a tough spot. We deployed V1.2 which still has the ELB's in place. However there is no upgrade path for V1.3 on AWS.

There is no addon for me to enable ELBs. What do I do now? Do I fork the repository and just stick on v1.2? Do I hand port in changes?

Having an ELB front the edge nodes is a good thing because we get health checks, also the ability to have those ELB's do SSL termination if we want.

@johntdyer -Might have input to this as well.

Basically I'm asking let's find a way to not leave AWS users hanging. How can we work together to solve this?
Thanks,

**Comment by stevendborrelli** [comment]
Until we figure out the cert issues, you can use remote modules to point to the old 1.2 elb configuration.

It would be something like this in your main.tf file at the root of the project:
```
module "elb" {
    source = https://github.com/CiscoCloud/mantl//tree/1.2.0/terraform/aws/elb
}
```
